src/taxonomy/api/hooks/api.js

```diff
@@ -5,16 +5,35 @@ import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 
 const getApiBaseUrl = () => getConfig().STUDIO_BASE_URL;
 const getTaxonomyListApiUrl = new URL('api/content_tagging/v1/taxonomies/?enabled=true', getApiBaseUrl()).href;
-
+/**
+  * @param {number} taxonomyId
+  */
+const getTaxonomyDetailApiUrl = (taxonomyId) => new URL(
+  `api/content_tagging/v1/taxonomies/${taxonomyId}/`,
+  getApiBaseUrl(),
+).href;
 /**
  * @returns {import("../types.mjs").UseQueryResult}
  */
 const useTaxonomyListData = () => (
   useQuery({
     queryKey: ['taxonomyList'],
     queryFn: () => getAuthenticatedHttpClient().get(getTaxonomyListApiUrl)
-      .then(camelCaseObject),
+      .then(camelCaseObject).then(({ data }) => data),
```

Comment:
@rpenido I think you need to change also the type that this function returns. You can run the types checks with `npm run types`